woudc_data_registry/models.py

```diff
@@ -78,18 +79,19 @@ class Country(base):
 
     __tablename__ = 'countries'
 
-    identifier = Column(String, nullable=False, primary_key=True)
```

Comment:
All the identifiers were renamed to be consistent across tables for easier joins (e.g. `deployment.station_id` matches with `station.station_id`). The ids were like that in all the archive tables and I found it really handy. I would be interested to hear the rationale for calling them all `identifier` since it looks like you want to keep it that way.